aiodownloader/downloader.py

```diff
@@ -158,3 +162,17 @@ async def download(self,
 
         logger.info('The download has been completed')
         return [task.result() for task in tasks]
+
+if __name__ == '__main__':
```

Comment:
This seems to be testing code. We probably don't want this here

Reply:
Hello, thanks for getting back to me. I couldn't figure out where github notifications were.
You are right about the raising the exception using a new class. Yes I should have done a pep8 check, so I'll change the none comparison.
About the testing code, I think it is a nice example code. Should it be in another file?

Regards